config/capz/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:main
+      - image: nhkregistry.azurecr.io/cluster-api-azure-controller:dev
```

Comment:
Will restore this to `gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:main`